src/taskparameters.ts

```diff
@@ -76,12 +76,14 @@ export class TaskParameters {
         this._location = core.getInput('location', { required: true });
         this._memory = parseFloat(core.getInput('memory'));
         this._containerName = core.getInput('name', { required: true });
+
         let osType = core.getInput('os-type');
-        if(osType != 'Linux' && 'Windows') {
-            throw Error('The Value of OS Type must be either Linux or Windows only!')
+        if (["Linux", "Windows"].includes(osType) == false) {
+            throw Error(`The Value of OS Type must be either Linux or Windows only - got ${osType}!`);
```

Comment:
You'll need to run the build command here to get these changes propagated to the lib/taskparameters.js files, since the action works with only js files

Reply:
Thanks for the response @mitsha-microsoft. I have run the build command.